test/Removed/BuildDir/SConstruct.kwarg

```diff
@@ -0,0 +1 @@
+SConscript('src/SConscript', build_dir='build')
```

Comment:
NOTE: the test for the removed `build_dir` keyword fails, because this call to `SConscript` with an unknown keyword apparently does *not* fail.  Should the test be removed? Should scons itself raise an error in this case? Or...?

Reply:
I'd create an issue that SConscript doesn't error when passed unrecognized arguments then add test.skip('SConscript doens't error with unknown arguments see GH Issue #..."